website/docs/core-concepts/connecting-to-data-sources/authentication/connect-to-apis.md

```diff
@@ -4,24 +4,168 @@ description: You can hit any REST endpoint available on the public internet
 
 # Configuring APIs
 
+Configuring APIs involves setting up and managing the various aspects of an API that allow it to function effectively. This can include defining the API's structure and capability, setting up authentication and authorization, and establishing connections with other systems or services.
+
 :::tip
-If your API is not available on the public internet, you must expose it via a service such as [https://ngrok.com/](https://ngrok.com/)
+If your API isn't available on the public internet, you must expose it via a service such as [https://ngrok.com/](https://ngrok.com/)
 :::
 
-## API Editor
+## API query editor
 
-The API pane is a REST interface that allows you to create and modify your existing APIs. All [REST HTTP methods](https://www.w3schools.in/restful-web-services/rest-methods) are supported and API values can be configured in the headers, params, and body fields
+The API pane is a REST interface that allows you to create and modify your existing APIs. All [REST HTTP methods](https://www.w3schools.in/restful-web-services/rest-methods) are supported and API values can be configured in the headers, Params, and body fields
 
 ![](</img/create_api_(1).gif>)
 
-## Importing CURL Commands
+## Configure API query
```

Comment:
Too many repetitions here. Can we have the common properties listed and then, for specific properties add the type of APIs?